src/bindings.cpp

```diff
@@ -69,6 +69,12 @@ void start_v8_isolate(void *dll){
   platformptr = v8::platform::CreateDefaultPlatform();
   v8::V8::InitializePlatform(platformptr);
 #endif
+
+#ifdef V8_ENABLE_SANDBOX
+  if (!v8::V8::InitializeSandbox()) {
+    REprintf("Error initializing the V8 sandbox\n");
+  }
+#endif
```

Comment:
This is no more:

Reply:
Thanks. Ill review asap after my vacation.

Reply:
No hurry, that was just a reminder for myself to update this PR. Enjoy your well deserved vakantie :)